activities/workshops/index.md

```diff
@@ -7,3 +7,4 @@ type: Index
 Here is a list of workshops developed or used by the Foundation for Public Code:
 
 * [Governance exercise](governance-exercise.md)
+* [Governance game](/governance-game/index.md)
```

Comment:
This link is broken, you probably mean this without the leading `/`

```suggestion
* [Governance game](governance-game/index.md)
```

Also, what is the difference between the Governance game and the Governance exercise?

Reply:
The governance exercise is the one we did at the Open day, and might still be a relevant workshop to have in store for the near future.